### PR TITLE
User must review TOS when updated

### PR DESCRIPTION
**Pull request by agustibr**
#### :tophat: What? Why?

When a user has accepted the TOS (Term Of Service), if it gets updated the user must review and accept again. Or an option to refuse.

#### :pushpin: Related Issues
- Related to Epic #3320
- Fixes #3318
- Relates to issue #3317

#### :clipboard: Subtasks
- [x] Add `CHANGELOG` entry
- [x] Add/modify seeds
- [x] Add tests
- [x] Add field `tos_accepted_at` to user
- [x] Added form to accept new TOS
- [x] Added option to refuse terms
- [x] signed_in user must accept to continue
- [x] fixed bug with `high_voltage` gem version
- [x] invited users should also accept TOS

### :camera: Screenshots (optional)

+ Redirected to tos page:
  screenshot from 2018-05-09 17-55-22

+ If TOS are accepted:
  screenshot from 2018-05-09 17-55-59

+ If the Refuse option is clicked a modal is shown with options:
  screenshot from 2018-05-16 16-49-05

+ Accept Invitation screen with TOS checkbox and text:
  screenshot from 2018-05-16 16-45-32

### CODE REVIEW

File: decidim-core/app/commands/decidim/create_omniauth_registration.rb [hunk 1]
```diff
@@ -59,6 +59,7 @@ def create_or_find_user
         @user.skip_confirmation! if verified_email
       end
 
+      @user.tos_accepted_at = @user.organization.tos_updated_at
```
Comment (tramuntanal) [inline decidim-core/app/commands/decidim/create_omniauth_registration.rb]: If I understand well, `@user.tos_accepted_at` is not the timestamp when the user accepted the tos but the version of the accepted tos. If this is the case, we should use another name like `@user.tos_version` or `@user.accepted_tos_version`.

**Review by tramuntanal — COMMENTED** [review]
Great work, just one doubt!

File: decidim-core/app/commands/decidim/create_omniauth_registration.rb (hunk 1 above)
Comment (agustibr) [reply decidim-core/app/commands/decidim/create_omniauth_registration.rb]: Yes you're right, `@user.tos_accepted_at` is the timestamp of the accepted version of the tos.
Maybe `@user.accepted_tos_updated_at` is more appropriate, as in [another PR](https://github.com/CodiTramuntana/decidim/pull/3) the organization will have the `tos_updated_at` attribute:

    add_column :decidim_organizations, :tos_updated_at, :datetime

Comment (agustibr) [reply decidim-core/app/commands/decidim/create_omniauth_registration.rb]: I've changed it to `accepted_tos_version` for `User` and to `tos_version` for `Organization`

**Pull request closed without merging**